test/helpers.js

```diff
@@ -0,0 +1,76 @@
+import test from 'ava';
```

Comment:
But in this case I create unit-test for each of them. And that make more easier to find what's wrong if main tests failed. 
If main test failed, but unit - not --- first should check code not covered with tests.